src/node_dotenv.cc

```diff
@@ -107,13 +107,28 @@ Local<Object> Dotenv::ToObject(Environment* env) const {
 
 std::string_view trim_spaces(std::string_view input) {
   if (input.empty()) return "";
-  if (input.front() == ' ') {
-    input.remove_prefix(input.find_first_not_of(' '));
+
```

Comment:
Can you add a comment to what this function does after your changes?

Reply:
sure 👍

Reply:
added, please let me know if the comment works for you 🙂